AAI/AAIConnectProfile.md

```diff
@@ -338,12 +338,60 @@ skinparam componentStyle rectangle
 left to right direction
 
 package "Unspecified clients, additional services, protocols" {
-component "<b>Visa Assertion Source</b> (1)\norganisation" as VisaSource1
-component "<b>Visa Assertion Source</b> (2)\norganisation" as VisaSource2
-component "<b>Visa Assertion Source</b> (...)\norganisation" as VisaSourceN
+component "<b>Visa Assertion Source</b> (1)\norganization" as VisaSource11
+component "<b>Visa Assertion Source</b> (2)\norganization" as VisaSource12
+component "<b>Visa Assertion Source</b> (1)\norganization" as VisaSource21
```

Comment:
Stay tuned. We discussed a simpler approach to this diagram that I am drafting now. Obviously it needs to make sense to you so we'll discuss more.

Reply:
I think the numbering is okay in the new set of diagrams but it could change if you prefer. Now it's 
- Example 1  - VAS 1  and 2
- Example 2 - VAS 1 and 2 and …

But if you think it's clearer it could change to 3, 4, …